Tests/LibUnicode/TestSegmenter.cpp

```diff
@@ -136,10 +136,10 @@ TEST_CASE(out_of_bounds)
         auto segmenter = Unicode::Segmenter::create(Unicode::SegmenterGranularity::Word);
         segmenter->set_segmented_text(text);
 
-        auto result = segmenter->previous_boundary(text.byte_count());
+        auto result = segmenter->previous_boundary(text.byte_count() + 1);
```

Comment:
Let's not remove testing for what happens when we pass in `text.byte_count()`. Let's be sure to test passing in both `text.byte_count()` and `text.byte_count() + 1`.